lib/compat/wordpress-6.0/class-gutenberg-rest-edit-site-export-controller.php

```diff
@@ -76,7 +76,10 @@ public function export() {
 			return $filename;
 		}
 
-		$theme_name = wp_get_theme()->get( 'TextDomain' );
+		$stylesheet             = get_stylesheet();
+		$stylesheet_directories = explode( '/', $stylesheet );
```

Comment:
Pardon my ignorance and for jumping in here, but is there any reason why `basename` couldn't be used instead?

Reply:
Good point. It might be even better because `basename` can handle both slash (/) and backslash (\) which often is an issue on Windows machines.

Reply:
Good idea, done here: https://github.com/WordPress/gutenberg/pull/41058/files